commerce/domain/tax/cart/calculator/classes/TaxCartCalculatorSampleTest.cls

```diff
@@ -0,0 +1,190 @@
+/**
+ * @description A Sample unit test for TaxCartCalculatorSample.
+ */
+@IsTest
+public class TaxCartCalculatorSampleTest {
```

Comment:
Are we missing tests for calculating taxes for Charge cartItems and for Adjustments?

Reply:
Covered adjustment & promotions in cartItem, now code coverage is 72%, not able to bring to 80% because there are some part of code can not be invoked from test class like `getTaxesFromExternalService`. 
<img width="1472" alt="Screenshot 2024-05-16 at 1 12 44 PM" src="https://github.com/bsrilok/commerce-extensibility/assets/163102218/3e4fb568-e91a-480e-b0d3-310177f01cb4">

Reply:
Is there a way to increase test coverage? `getTaxesFromExternalService`  is 27 lines of code. That can’t affect 28% of code coverage.

Reply:
Added few more test cases & refactored the code by deleting dead code present in apex class now cc is 80%
<img width="1728" alt="Screenshot 2024-05-21 at 3 19 01 PM" src="https://github.com/bsrilok/commerce-extensibility/assets/163102218/b02465c0-9d28-4dc5-96e2-33914718112c">